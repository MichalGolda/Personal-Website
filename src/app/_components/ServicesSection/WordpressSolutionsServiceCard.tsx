import ServiceCard from "./ServiceCard";
import ServiceIcon from "./ServiceIcon";

const WordpressServiceIcon = () => (
  <ServiceIcon src="/wordpress (services-section).webp" alt="wordpress" />
);

const WordpressSolutionsServiceCard = () => (
  <ServiceCard
    title="Rozwiązania oparte o WordPress CMS"
    content={
      <>
        Tworzenie niestandardowych szablonów, dostosowanych do indywidualnych
        potrzeb i wizji klienta. <br />
        <br />
        Konfiguracja wtyczek i optymalizacja funkcjonalnośći i wydajnośći
        strony. <br />
        <br />
        Modernizacja istniejących stron opartych na WordPress, w tym
        aktualizacja szablonów, wtyczek i optymalizacja wydajności. <br />
        <br />
        Migracja ze starej platformy na WordPress, zapewniając bezpieczne
        przeniesienie danych i zachowanie integralności strony.
      </>
    }
    icon={<WordpressServiceIcon />}
    data-aos="fade-right"
    data-aos-duration="500"
  />
);

export default WordpressSolutionsServiceCard;
