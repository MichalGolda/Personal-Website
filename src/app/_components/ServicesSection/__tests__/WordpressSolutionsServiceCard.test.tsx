import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import WordpressSolutionsServiceCard from "../WordpressSolutionsServiceCard";

describe("WordpressSolutionsServiceCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<WordpressSolutionsServiceCard />);

    expect(container).toMatchSnapshot();
  });
});
