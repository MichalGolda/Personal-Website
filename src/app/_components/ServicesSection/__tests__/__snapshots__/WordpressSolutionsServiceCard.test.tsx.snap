// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordpressSolutionsServiceCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4"
    data-aos="fade-right"
    data-aos-duration="500"
  >
    <div
      class="flex flex-row gap-x-4 md:flex-col md:gap-y-2 lg:flex-row md:items-start lg:items-center items-center"
    >
      <img
        alt="wordpress"
        class="w-[48px] lg:w-[64px]"
        data-nimg="1"
        decoding="async"
        height="64"
        loading="lazy"
        src="/_next/image?url=%2Fwordpress%20(services-section).webp&w=128&q=75"
        srcset="/_next/image?url=%2Fwordpress%20(services-section).webp&w=64&q=75 1x, /_next/image?url=%2Fwordpress%20(services-section).webp&w=128&q=75 2x"
        style="color: transparent;"
        width="64"
      />
      <h4
        class="font-bold text-lg md:text-xl lg:text-2xl text-secondary"
      >
        Rozwiązania oparte o WordPress CMS
      </h4>
    </div>
    <p
      class="text-body text-sm md:text-base"
    >
      Tworzenie niestandardowych szablonów, dostosowanych do indywidualnych potrzeb i wizji klienta. 
      <br />
      <br />
      Konfiguracja wtyczek i optymalizacja funkcjonalnośći i wydajnośći strony. 
      <br />
      <br />
      Modernizacja istniejących stron opartych na WordPress, w tym aktualizacja szablonów, wtyczek i optymalizacja wydajności. 
      <br />
      <br />
      Migracja ze starej platformy na WordPress, zapewniając bezpieczne przeniesienie danych i zachowanie integralności strony.
    </p>
    <span
      class="block bg-primary w-32 h-0.5"
    />
  </div>
</div>
`;
